setup.py

```diff
@@ -165,8 +165,8 @@ def run(self):
         },
         entry_points={
             'console_scripts': [
-                'mxnet-model-server=mms.model_server:start',
-                'mxnet-model-export=mms.export_model:main'
+                'multi-model-server=mms.model_server:start',
+                'multi-model-export=mms.export_model:main'
```

Comment:
`mxnet-model-export`  isn't required anymore. mxnet-model-export was the original name of model-archiver. We have it here for backeward compatibility.